### PR TITLE
Ch7 editor's review

### PR DESCRIPTION
**Pull request by jni**

### CODE REVIEW

File: markdown/ch7.markdown [hunk 1]
```diff
@@ -144,29 +150,50 @@ to search for optimal parameters:
...
 print(f'The optimal shift for correction is: {res.x}')
 ```
 
-Brilliant! We shifted it by +50 pixels, and, thanks to our MSE measure, SciPy's
+It worked! We shifted it by +50 pixels, and, thanks to our MSE measure, SciPy's
 `optimize.minimize` function has given us the correct amount of shift (-50) to
 get it back to its original state.
 
-Unfortunately, this brings us to the principal difficulty of this kind of
+It turns out, however, that this was the simplest kind of optimization, and
```
Comment (stefanv) [inline markdown/ch7.markdown]: What does "simplest kind of optimization" mean?

File: markdown/ch7.markdown [hunk 2]
```diff
@@ -401,24 +430,41 @@ for ax in (ax0, ax1, ax2):
...
-Oops! Even though we started closer to the original image, we failed to
+Even though we started closer to the original image, we failed to
 recover the correct rotation. This is because optimization techniques can get
 stuck in local minima, little bumps on the road to success, as we saw above
 with the shift-only alignment. They can therefore be quite sensitive to the
 starting parameters.
 
+## Avoiding local minima with basin hopping
+
+A 1997 algorithm devised by David Wales and Jonathan Doyle [^basinhop], called
```
Comment (stefanv) [inline markdown/ch7.markdown]: I wish I knew this during my PhD; I had recurring issues with registration that messed up many experiments to the point of exasperation.

File: markdown/ch7.markdown [hunk 3]
```diff
@@ -479,23 +524,28 @@ for ax in (ax0, ax1, ax2):
     ax.axis('off')
 ```
 
-Boom! Consider that basin *hopped!*
+Success! Basin-hopping was able to recover the correct alignment, even in the
```
Comment (stefanv) [inline markdown/ch7.markdown]: I understand why this change was necessary... but... it was funny ;)

File: markdown/ch7.markdown [hunk 4]
```diff
@@ -479,23 +524,28 @@ for ax in (ax0, ax1, ax2):
...
-with a different color filter placed in front of the lens.
+excellent. But it turns out that we've only solved the easiest of registration
+problems: aligning images of the same *modality*. This means that we expect
+bright pixels in the reference image to match up to bright pixels in the test
+image.
+
+We now move on to aligning different color channels of the same image,
+where we can no longer rely on the channels having the same modality. This task
+has historical significance: between 1909 and 1915, the photographer Sergei
+Mikhailovich Prokudin-Gorskii produced color photographs of the Russian empire
```
Comment (stefanv) [inline markdown/ch7.markdown]: I'm super excited that this is in the book!!

Comment (jni) [inline markdown/ch7.markdown]: Yeah it's a beautiful example. =)

**Comment by jni** [comment]
@hdashnow @stefanv this should be ready to go then — I rephrased "simplest kind of optimization".

**Comment by hdashnow** [comment]
Looks good to me! I'll let @stefanv merge in case he has any further comments.

**Comment by stefanv** [comment]
In it goes!
